Jamulus.pro

```diff
@@ -95,20 +95,28 @@ win32 {
     contains(CONFIG, "jackonwindows") {
         message(Using Jack instead of ASIO.)
 
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
-        }
+        contains(QT_ARCH, "x86") {
```

Comment:
Has this been tested?
I'd assume that `contains(x86_64, x86)` and `contains(x86, x86)` will both be true?
[`equals()`](https://doc.qt.io/qt-5/qmake-test-function-reference.html#equals-variablename-value) looks more correct to me for this use case.

Reply:
> I'd assume that `contains(x86_64, x86)` and `contains(x86, x86)` will both be true?

Interesting spot! I have checked, and while it was working on my system when I compiled it...it sounds as if it was just working by luck.

When the build is performed on windows for 64-bit, the QT_ARCH is set to "x86_64".
When the build is performed on windows for 32-bit, the QT_ARCH is set to "i386"

So the jamulus.pro needs updating for this.

Reply:
What about Windows 64 bits in 32 bits mode ? WOW64 let the user work in 32 bits mode. Is the QT_ARCH set to 'x86_64' in this case ?

That's why I wanna add a target first with win64 label... ;-)

Reply:
Think it's ok in the last commit:
5aa0168